src/components/LoanCards/LoanCardController.vue

```diff
@@ -13,12 +13,15 @@
 		:is-funded="isFunded"
 		:is-selected-by-another="isSelectedByAnother"
 		:amount-left="amountLeft"
+		:show-category-description="showCategoryDescription"
```

Comment:
We don't need to include this here...it's shouldn't be used in the loan card view.